package.json

```diff
@@ -90,7 +90,7 @@
     "Panagiotis Tourlas <panagiot_tourlov@hotmail.com>"
```

Comment:
Feel free to add yourself as an author here

Reply:
Thank you very much. I added myself here.